src/main/java/com/amazonaws/kinesisvideo/auth/DefaultAuthCallbacks.java

```diff
@@ -41,7 +41,7 @@
     /**
      * Used for logging
      */
-    private final Log log;
+    private final Logger logger;
```

Comment:
Any reason not to name the variable `log` then you don't have to make a lot of those changes below. Seems more natural to be called log.something instead of logger.

Reply:
No specific reason. I just did that for uniformity. Should I change it to `log`? It is `logger` everywhere currently

Reply:
Updated this in the recent commit